include/vapor/IsoSurfaceRenderer.h

```diff
@@ -22,8 +22,8 @@ class RENDER_API IsoSurfaceRenderer : public RayCaster
     }
 
 protected:
-    void _loadShaders();
-    void _3rdPassSpecialHandling( bool );
+    int  _loadShaders();
+    void _3rdPassSpecialHandling( bool, long );
```

Comment:
Please give the arguments names

Reply:
The argument name is given in the .cpp file. Does this address your concern?

Reply:
No because you reference the header file for function definitions.

Reply:
Agreed

Reply:
fixed in 94798d0f